### ISSUE TITLE
When submitting the answers there should be validation?

### ISSUE THREAD

**Issue by ayushcs** [ayushcs/slambook#4]
Steps to produce
1) Open this link
https://ayushcs.github.io/slambook/#/users/id=dWNhc3Z8fEF5dXNoMTIz

2) write any answer and press submit.
3) in dialog press the button without writing anything

Issue 
There should be validation

**Comment by nihalkza** [comment]
yes ! I knew it during the work, I am working on it. will update here after done.

**Comment by nihalkza** [comment]
code committed & Committed new UI in Create page. also code Quality.
